packages/opencensus-core/test/test-console-exporter.ts

```diff
@@ -75,12 +63,11 @@ describe('ConsoleLogExporter', () => {
     it('should end a span', () => {
       const exporter = new ConsoleExporter(defaultBufferConfig);
       tracer.registerSpanEventListener(exporter);
-      // const rootSpan = new RootSpan(tracer);
-      const rootSpans = createRootSpans();
-      for (const rootSpan of rootSpans) {
+
+      tracer.startRootSpan({name: 'testRootSpans'}, rootSpan => {
         rootSpan.end();
-      }
-      assert.ok(true);
+        assert.ok(exporter.buffer.getQueue().length > 0);
```

Comment:
Is this testing the tracer, the exporter buffer, or the exporter? It's not clear which one, but it doesn't seem like it is really testing functionality that is unique to `ConsoleLogExporter`.

Reply:
This is just to check if the span was added to the buffer, that should be the proper behaviour of `onEndSpan()`. I also added a few more assertions to make sure the span in the exporter buffer is right.